### PR TITLE
Allow to declare quantity and cursor when retrieving broadcaster's subscribers

### PR DESCRIPTION
**Pull request by pietrodicaprio**
This change allows declaring a quantity (`first`) of subscribers to retrieve and the `cursor` when calling `GetBroadcasterSubscriptions`

### CODE REVIEW

File: TwitchLib.Api.Helix/TwitchLib.Api.Helix.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>TwitchLib.Api.Helix</PackageId>
-    <Version>3.1.2</Version>
+    <Version>3.2.1-SP</Version>
```
Comment (Syzuna) [inline TwitchLib.Api.Helix/TwitchLib.Api.Helix.csproj:6]: Please dont change around the version

File: TwitchLib.Api.Helix/TwitchLib.Api.Helix.csproj [hunk 2]
```diff
@@ -17,8 +17,8 @@
     <RepositoryType>Git</RepositoryType>
     <PackageTags>twitch library irc chat c# csharp api events pubsub net standard 2.0</PackageTags>
     <NeutralLanguage>en-US</NeutralLanguage>
-    <AssemblyVersion>3.1.2</AssemblyVersion>
-    <FileVersion>3.1.2</FileVersion>
+    <AssemblyVersion>3.2.1</AssemblyVersion>
```
Comment (Syzuna) [inline TwitchLib.Api.Helix/TwitchLib.Api.Helix.csproj:20]: Please dont change around the version

File: TwitchLib.Api.Helix/TwitchLib.Api.Helix.csproj [hunk 3]
```diff
@@ -17,8 +17,8 @@
...
+    <FileVersion>3.2.1</FileVersion>
```
Comment (Syzuna) [inline TwitchLib.Api.Helix/TwitchLib.Api.Helix.csproj:21]: Please dont change around the version

File: TwitchLib.Api.Helix/Subscriptions.cs [hunk 4]
```diff
@@ -37,7 +37,8 @@ public Task<GetUserSubscriptionsResponse> GetUserSubscriptionsAsync(string broad
             return TwitchGetGenericAsync<GetUserSubscriptionsResponse>("/subscriptions", ApiVersion.Helix, getParams, accessToken);
         }
 
-        public Task<GetBroadcasterSubscriptionsResponse> GetBroadcasterSubscriptions(string broadcasterId, string accessToken = null)
+        public Task<GetBroadcasterSubscriptionsResponse> GetBroadcasterSubscriptions(string broadcasterId, string cursor = null,
```
Comment (Syzuna) [inline TwitchLib.Api.Helix/Subscriptions.cs:40]: to keep things consistent I would like to have quantity changed to first as variable name

**Review by Syzuna — CHANGES_REQUESTED** [review]

**Comment by Syzuna** [comment]
also please dont PR against the master branch.
use the dev branch instead

**Review by pietrodicaprio — COMMENTED** [review]
Got it. Sorry for the version naming, initially was not meant for a PR but then I said "why not?".
Going to take care for sure for any new change I do

**Comment by Syzuna** [comment]
LGTM. will merge.

Thanks and sorry for keeping you waiting.
The last months were kind of a ride and its still ongoing.

**Review by Syzuna — APPROVED** [review]

**Pull request merged**